node/src/accountant/scanners_tools.rs

```diff
@@ -0,0 +1,539 @@
+            balance: debt,
+            last_paid_timestamp: from_time_t(time),
+            pending_payable_opt: None,
+        };
+
+        let result = is_payable_qualified(now, &unqualified_payable_account, &payment_thresholds);
+
+        assert_eq!(result, None);
+    }
+
+    #[test]
+    fn payable_above_threshold_values_is_marked_qualified_and_returns_threshold() {
+        let now = SystemTime::now();
+        let payment_thresholds = PaymentThresholds::default();
+        let debt = payment_thresholds.permanent_debt_allowed_gwei + 1_000_000_000;
+        let time = to_time_t(now) - payment_thresholds.maturity_threshold_sec - 1;
+        let payment_thresholds_rc = Rc::new(payment_thresholds);
+        let qualified_payable = PayableAccount {
+            wallet: make_wallet("wallet0"),
+            balance: debt,
+            last_paid_timestamp: from_time_t(time),
+            pending_payable_opt: None,
+        };
+        let threshold = calculate_payout_threshold(
+            payable_time_diff(now, &qualified_payable),
+            &payment_thresholds_rc,
+        );
+        eprintln!("Threshold: {}, Debt: {}", threshold, debt);
+
+        let result = is_payable_qualified(now, &qualified_payable, &payment_thresholds_rc);
+
+        assert_eq!(result, Some(threshold as u64));
+    }
+
+    #[test]
+    fn qualified_payables_can_be_filtered_out_from_non_pending_payables_along_with_their_summary() {
+        let now = SystemTime::now();
+        let payment_thresholds = PaymentThresholds::default();
+        let unqualified_payable_accounts = vec![PayableAccount {
+            wallet: make_wallet("wallet1"),
+            balance: payment_thresholds.permanent_debt_allowed_gwei + 1,
+            last_paid_timestamp: from_time_t(
+                to_time_t(now) - payment_thresholds.maturity_threshold_sec + 1,
+            ),
+            pending_payable_opt: None,
+        }];
+        let qualified_payable_accounts = vec![
+            PayableAccount {
+                wallet: make_wallet("wallet2"),
+                balance: payment_thresholds.permanent_debt_allowed_gwei + 1_000_000_000,
+                last_paid_timestamp: from_time_t(
+                    to_time_t(now) - payment_thresholds.maturity_threshold_sec - 1,
+                ),
+                pending_payable_opt: None,
+            },
+            PayableAccount {
+                wallet: make_wallet("wallet3"),
+                balance: payment_thresholds.permanent_debt_allowed_gwei + 1_200_000_000,
+                last_paid_timestamp: from_time_t(
+                    to_time_t(now) - payment_thresholds.maturity_threshold_sec - 100,
+                ),
+                pending_payable_opt: None,
+            },
+        ];
+        let mut all_non_pending_payables = Vec::new();
+        all_non_pending_payables.extend(qualified_payable_accounts.clone());
+        all_non_pending_payables.extend(unqualified_payable_accounts.clone());
```

Comment:
Interestingly, this whole big block of code above reassembles the utility function which I saw somewhere else...probably in `accountant/test_utils.rs` at its bottom. Please, because you already have it, reuse it here again.